content/en/docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -72,7 +72,7 @@ the Pod:
 
     ```shell
     root@redis:/data/redis# apt-get update
-    root@redis:/data/redis# apt-get install procps
+    root@redis:/data/redis# apt-get install -y procps
```

Comment:
Please remove the `-y`. Admins can add that if they want, but we don't want to guide them to force things to install without examining the dependencies that will also be installed.

/hold

Reply:
@mistyhacks  Thanks.